test/server/admin/stats_handler_speed_test.cc

```diff
@@ -65,17 +65,6 @@ class StatsHandlerTest {
 } // namespace Server
 } // namespace Envoy
 
-class UseRe2Filters {
-public:
-  UseRe2Filters(bool use_re2) {
-    scoped_runtime_.mergeValues(
-        {{"envoy.reloadable_features.admin_stats_filter_use_re2", use_re2 ? "true" : "false"}});
```

Comment:
since no longer using the scoped runtime can we remove the include?

Reply:
Done